examples/fineFoods/admin/mui/src/components/store/storeProducts.tsx

```diff
@@ -0,0 +1,300 @@
+import React from "react";
+import { useTranslate, useTable, useUpdate } from "@pankod/refine-core";
+import { useModalForm } from "@pankod/refine-react-hook-form";
+import {
+    Grid,
+    Paper,
+    Typography,
+    Box,
+    InputBase,
+    IconButton,
+    Stack,
+    Pagination,
+    Button,
+    Modal,
+    Fade,
+} from "@pankod/refine-mui";
+import { SearchOutlined, CloseOutlined } from "@mui/icons-material";
+
+import {
+    CategoryFilter,
+    ProductItem,
+    CreateProduct,
+    EditProduct,
+} from "components";
+import { IStore, IProduct } from "interfaces";
+
+type StoreProductsProps = {
+    record: IStore;
+    close: () => void;
+    visible: boolean;
+};
+
+export const StoreProducts: React.FC<StoreProductsProps> = ({
+    record,
+    close: modalClose,
+    visible: modalVisible,
+}) => {
+    const t = useTranslate();
+
+    const {
+        modal: { visible, close, show, saveButtonProps },
+        register,
+        refineCore: { onFinish },
+        formState: { errors },
+        handleSubmit,
+        watch,
+        setValue,
+        control,
+    } = useModalForm<IProduct>({
+        refineCoreProps: { action: "create" },
+    });
+
+    const {
+        modal: {
+            visible: visibleEditDrawer,
+            close: closeEditDrawer,
+            show: showEditDrawer,
+            saveButtonProps: editSaveButtonProps,
+        },
+        refineCore: { onFinish: editOnFinish },
+        register: editRegister,
+        formState: { errors: editErrors },
+        handleSubmit: editHandleSubmit,
+        watch: editWatch,
+        setValue: editSetValue,
+        control: editControl,
+    } = useModalForm<IProduct>({
+        refineCoreProps: { action: "edit" },
```

Comment:
```suggestion
        refineCoreProps: { action: "edit" },
```
In the Ant Design version, we have given redirect as false. No need here?
https://github.com/pankod/refine/blob/master/examples/fineFoods/admin/src/components/store/storeProducts.tsx#L112